openc3/lib/openc3/io/json_api.rb

```diff
@@ -0,0 +1,71 @@
+# encoding: ascii-8bit
+
+# Copyright 2023 OpenC3, Inc.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# This file may also be used under the terms of a commercial license
+# if purchased from OpenC3, Inc.
+
+require 'openc3/io/json_api_object'
+
+module OpenC3
+  class JsonApi
+    # Create a JsonApiObject connection to the API server
+    def initialize(microservice_name: 'CFDP', prefix: '/cfdp', schema: 'http', hostname: nil, port: 2905, timeout: 5.0, url: nil, scope: $openc3_scope)
```

Comment:
Did you mean to check this in? What are these defaults?

Reply:
I did sneak that file in.  But you're write I should cleanup the defaults